### PR TITLE
Added √ to mocha passed test regex

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
The Agent suddenly changed encoding for an existing run and the passed test cases symbol changed. This is a reactionary change to that

### CODE REVIEW

File: Agent.Plugins.Log.TestResultParser.Parser/Parser/Node/Mocha/MochaRegexes.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class MochaRegexes
         /// Matches lines with the following regex:
         /// ^(  )+((ΓêÜ)|✓) (((.+) \\(([0-9]+)(ms|s|m|h)\\)$)|(.+)$)
         /// </summary>
-        public static Regex PassedTestCase { get; } = new Regex($"^(  )+((ΓêÜ)|✓) (((?<{RegexCaptureGroups.TestCaseName}>.+) \\((?<{RegexCaptureGroups.TestRunTime}>[0-9]+)(?<{RegexCaptureGroups.TestRunTimeUnit}>ms|s|m|h)\\)$)|(?<{RegexCaptureGroups.TestCaseName}>.+)$)", RegexOptions.ExplicitCapture);
+        public static Regex PassedTestCase { get; } = new Regex($"^(  )+((ΓêÜ)|✓|√) (((?<{RegexCaptureGroups.TestCaseName}>.+) \\((?<{RegexCaptureGroups.TestRunTime}>[0-9]+)(?<{RegexCaptureGroups.TestRunTimeUnit}>ms|s|m|h)\\)$)|(?<{RegexCaptureGroups.TestCaseName}>.+)$)", RegexOptions.ExplicitCapture);
```
Comment (nigurr) [inline Agent.Plugins.Log.TestResultParser.Parser/Parser/Node/Mocha/MochaRegexes.cs]: Can we check for other parsers too?

Comment (ShreyasRmsft) [reply Agent.Plugins.Log.TestResultParser.Parser/Parser/Node/Mocha/MochaRegexes.cs]: ok will do a check

Comment (ShreyasRmsft) [reply Agent.Plugins.Log.TestResultParser.Parser/Parser/Node/Mocha/MochaRegexes.cs]: this is the sqrt symbol fyi

Comment (ShreyasRmsft) [reply Agent.Plugins.Log.TestResultParser.Parser/Parser/Node/Mocha/MochaRegexes.cs]: Other parses seem good for now.

**Pull request merged**